src/splitInteger.test.js

```diff
@@ -4,18 +4,24 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(17, 4))
+    .toHaveLength(4);
```

Comment:
u test the length of an array, so add it as separate test, because here u "should split a number into equal parts
  if a value is divisible by a numberOfParts"